app/models/cohort_columns/base.rb

```diff
@@ -22,6 +22,15 @@ class Base < ::ModelForm
     attribute :cohort_client
     attribute :editable, Boolean, lazy: false, default: true
     attribute :current_user
+    attribute :cohort_column, GrdaWarehouse::Cohorts::CohortColumn, lazy: true, default: ->(r, _) { GrdaWarehouse::Cohorts::CohortColumn.find_by!(class_name: r.class_name) }
+
+    def active?
+      cohort_column&.active?
+    end
+
+    def class_name
+      self.class.name
```

Comment:
Not sure this matters, but if we're kinda sorta using these as STI models, this should probably be:
```suggestion
      self.class.sti_name
```

Reply:
Actually, never mind.  After looking at the [source](https://apidock.com/rails/v7.1.3.4/ActiveRecord/Inheritance/ClassMethods/sti_name) of `sti_name`, I don't think this is necessary at this time.  I was under the impression it was doing something more to ensure it was DB safe or something.